lib/rubocop/cop/rails/http_status.rb

```diff
@@ -0,0 +1,181 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Enforces use of symbolic or numeric value to define HTTP status.
+      #
+      # @example `EnforcedStyle: symbolic` (default)
+      #   # bad
+      #   render :foo, status: 200
+      #   render json: { foo: 'bar' }, status: 200
+      #   render plain: 'foo/bar', status: 304
+      #   redirect_to root_url, status: 200
```

Comment:
[nit] Probably the status should be `301` as the good example is `status: :moved_permanently`.

Reply:
Good catch, I missed that one !